tools/setup.py

```diff
@@ -46,10 +45,12 @@
     namespace_packages=[
     ],
     scripts=[
-        "check_style",
-        "check_style.bat",
-        "run_tests",
-        "run_tests.bat"
+        "scripts/check_style",
+        "scripts/check_style.bat",
+        "scripts/run_tests",
+        "scripts/run_tests.bat",
+        'scripts/azt',
+        'scripts/azt.bat'
```

Comment:
Consider renaming this to `azdev` instead of `azt`?
Then we can use this CLI for other dev tools. I assume t stood for Test.

Reply:
ok.